tests/api/test_parameters.py

```diff
@@ -456,11 +456,11 @@ def test_args_kwargs(simple_app):
     app_client = simple_app.test_client()
     resp = app_client.get("/v1.0/query-params-as-kwargs")
     assert resp.status_code == 200
-    assert resp.json() == {}
+    assert resp.json() == {"body": {}}
 
     resp = app_client.get("/v1.0/query-params-as-kwargs?foo=a&bar=b")
     assert resp.status_code == 200
-    assert resp.json() == {"foo": "a"}
+    assert resp.json() == {"foo": "a", "body": {}}
```

Comment:
Not 100% sure about changing these tests this way.
I believe these view functions use `**kwargs`, so the `body` gets passed in here - so in that sense it seems logical. On the other hand, if the endpoint doesn't define a body, it's weird that it will be put in the `kwargs` (although I would think it currently also happens for `POST` methods and the like)

Reply:
Agree that it's not ideal. For `POST` this is probably more expected than for `GET`. I'm wondering if we can add a check so we only pass in the body if it's present, but I believe it's hard to discern between no body and an empty body. So then we'd probably have to check if the spec defines one.

Reply:
I am now checking whether the operation has defined a request body, and only pass the body to the view function in case it has.